Cargo.toml

```diff
@@ -29,11 +29,20 @@ route-recognizer = "0.1.12"
 serde = "1.0.90"
 serde_derive = "1.0.90"
 serde_json = "1.0.39"
+serde_urlencoded = "0.5.5"
 slog = "2.4.1"
 slog-async = "2.3.0"
 slog-term = "2.4.0"
 typemap = "0.3.3"
-serde_urlencoded = "0.5.5"
+
+[dependencies.accept-encoding]
+optional = true
+git = "https://github.com/fairingrey/accept-encoding"
```

Comment:
Should we move to rustasync?

Reply:
It's there, I'm just waiting for yoshua to review and merge https://github.com/rustasync/accept-encoding/pull/2